src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -42,9 +42,10 @@ class TaskBuilderTest extends MarathonSpec {
       )
     )
 
-    assert(task.isDefined)
+    assert(task.isInstanceOf[TaskBuilder.BuildSuccess])
```

Comment:
This could read more nicely as `task.isSuccess`, see comments on `TaskBuilder.BuildResult` above.